src/main/java/utility/PropertiesLoader.java

```diff
@@ -0,0 +1,79 @@
+package utility;
+
+import com.google.common.base.Strings;
+import lombok.SneakyThrows;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+
+import static java.util.Objects.*;
+
+public class PropertiesLoader {
+
+    private static final String PROPERTIES_FILE = "/properties/application.properties";
```

Comment:
This constant is used only once. Consider to inline it

Reply:
I know that "rule" says if we use it once make it inline. But I think it is better to make such a field 1st for visibility, and 2nd in terms of scaling and reusing.